python/nav/web/maintenance/views.py

```diff
@@ -252,9 +252,7 @@ def cancel(request, task_id):
 def edit(request, task_id=None, start_time=None, **_):
     account = get_account(request)
     quickselect = QuickSelect(service=True)
-    component_trail = None
-    component_keys = None
-    task = None
+    component_trail = component_keys_errors = component_data = task = None
```

Comment:
It probably would be cleaner if you None'd everything here, in addition:

* component_keys
* component_data_errors

Reply:
@hmpf None of these are referenced in any place where they might be uninitialized, so I didn't bother.  In fact, I removed one variable from this list that was initialized unnecessarily to `None`.

My IDE even clearly marks which variables are potentially used before assignment, so this didn't require much thought at all.

Reply:
Ah, I prefer not to make assumptions about IDEs. Approved, then.

Reply:
You don't need to make assumptions about IDE's either.

This is part of the IDE's linting rules, and should be caught by most linters, including SonarCloud (IMNSHO).  However, since NAV is older than most of our attempts at running automatic linters, we have largely elected to ignore the vast amounts of linter violation reports on the legacy codebase.  The reports we do see are usually on what actually was changed in PRs.